plugin/core/views.py

```diff
@@ -874,10 +952,8 @@ def format_completion(
 ) -> sublime.CompletionItem:
     # This is a hot function. Don't do heavy computations or IO in this function.
     item_kind = item.get("kind")
-    if isinstance(item_kind, int) and 1 <= item_kind <= len(COMPLETION_KINDS):
-        kind = COMPLETION_KINDS[item_kind - 1]
-    else:
-        kind = sublime.KIND_AMBIGUOUS
+    kind = COMPLETION_KINDS[item_kind] if isinstance(item_kind, int) and item_kind in COMPLETION_KINDS \
```

Comment:
Would it be possible to avoid the double dict lookup for the key here? We do a lookup for the `item_kind` key in `item_kind in COMPLETION_KINDS` as test, and then do the same lookup again in `COMLETION_KINDS[item_kind]`. Perhaps use something like `kind = COMPLETION_KINDS.get(item_kind)` and test for a `None` value, which is just **1** lookup in the dict.